pkg/instrumentation/helper.go

```diff
@@ -171,3 +171,13 @@ func setContainersFromAnnotation(inst *instrumentationWithContainers, annotation
 	inst.Containers = append(inst.Containers, languageContainers...)
 	return nil
 }
+
+func setImagePullPolicy(instSpec corev1.PullPolicy) corev1.PullPolicy {
```

Comment:
Is this necessary? Can't we just leave this empty if the user does?

Reply:
I will have a look whether we could do it.